webbpsf/webbpsf_core.py

```diff
@@ -280,6 +280,205 @@ def detector_position(self, position):
         self._detector_position = (int(position[0]),int(position[1]))
 
 
+    #----- actual optical calculations follow here -----
+    def calcPSF(self, outfile=None, source=None, filter=None, nlambda=None, monochromatic=None,
+                fov_arcsec=None, fov_pixels=None,  oversample=None, detector_oversample=None,
```

Comment:
I verified that this residual copy of calcPSF was superfluous, so the webbpsf_core.py is now synced with the master branch.